src/viam/sdk/components/movement_sensor/client.cpp

```diff
@@ -26,90 +27,71 @@ MovementSensorClient::MovementSensorClient(std::string name, std::shared_ptr<grp
 using namespace viam::component::movementsensor::v1;
 
 Vector3 MovementSensorClient::get_linear_velocity(const AttributeMap& extra) {
-    GetLinearVelocityResponse response;
-    stub_wrapper<GetLinearVelocityRequest>(
-        this, response, extra, &MovementSensorClient::Stub::GetLinearVelocity);
-    return Vector3::from_proto(response.linear_velocity());
+    return make_client_helper(this, *stub_, &MovementSensorClient::Stub::GetLinearVelocity)
+        .with(extra)
+        .invoke([](auto& response) { return Vector3::from_proto(response.linear_velocity()); });
```

Comment:
```suggestion
        .invoke([&](auto& response) { return Vector3::from_proto(response.linear_velocity()); });
```
Why are these `[=]` lambdas when the ones on the motor are `[&]`?

Reply:
Because the response handler doesn't need access to any captured state, so we don't need `[&]`.  We need it on the `with` lambdas because we need access to the paramers to the function. I don't think motor does it any differently, and I'm pretty sure there aren't any `[=]` lambdas at all.

Reply:
Ah sounds good; I was seeing `[&]` captures in motor in `with` functions.

> and I'm pretty sure there aren't any [=] lambdas at all.

Oh I thought `[]` implicitly meant `[=]`.